packages/v4/src/content/contribute/design/design.md

```diff
@@ -31,4 +31,12 @@ __Example__
 
 To open an issue or browse existing issues, visit the [patternfly-design-kit repo](https://github.com/patternfly/patternfly-design-kit/) on GitHub.
 
+### Contributing new icons
+We encourage designers to work with the existing PatternFly icon set when choosing icons to use in their project. These include both custom and existing Font Awesome icons that cover most common use cases. To see what's available, vist our [Icons page](https://www.patternfly.org/v4/guidelines/icons). There may be occasions where you want to propose a new icon to be added to PatternFly.
```

Comment:
This reads a little weird. I would maybe add a "However," before "there may be occasions.."